testsuite/features/build_validation/reposync/srv_sync_all_products.feature

```diff
@@ -100,7 +100,7 @@ Feature: Synchronize products in the products page of the Setup Wizard
     Then I should see the "Development Tools Module 15 SP4 x86_64" selected
     When I select "Containers Module 15 SP4 x86_64" as a product
     Then I should see the "Containers Module 15 SP4 x86_64" selected
-    When I select or deselect "SUSE Manager Client Tools Beta for SLE 15 x86_64 (BETA)" beta client tools
+    When I select or deselect "SUSE Multi-Linux Manager Client Tools Beta for SLE 15 x86_64 (BETA)" beta client tools
```

Comment:
Not sure if I'm not following correctly this scenario, but I see you only add the new MLM beta client tools while you should add both beta and non beta

Reply:
the non-beta get added automaticall by adding the base product